SupportYourLocals.Data/XMLData.cs

```diff
@@ -0,0 +1,164 @@
+﻿using MapControl;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Security.Cryptography;
+using System.Text;
+using System.Xml;
+using System.Xml.Linq;
+
+namespace SupportYourLocals.Data
+{
+    public class XMLData : IDataStorage
+    {
+        private string filePath = @"./LocalSellersData.xml";
+        private string fileName = "LocalSellersData.xml";
+        XDocument doc = new XDocument();
+        XmlDocument document = new XmlDocument();
+        public XMLData()
+        {
+
+            doc = XDocument.Load(filePath);
+            if (!File.Exists(filePath))
+            {
+                doc.Save(filePath);
+            }
+            else
+            {
+                doc = XDocument.Load(filePath);
+            }
+        }
+
+        ~XMLData()
+        {
+            SaveData();
+        }
+
+        public void AddData(LocationData data)
+        {
+            // Add a local seller after clicking save
+            //XDocument doc = XDocument.Load(filePath);
+            XElement root = new XElement("LocalSeller");
+            root.Add(new XAttribute("ID", data.ID));
+            root.Add(new XAttribute("Location", data.Location));
+            root.Add(new XAttribute("Name", data.Name));
+            root.Add(new XAttribute("AddedByID", data.AddedByID));
+            root.Add(new XAttribute("Time", data.Time));
+            addProductTypesToXml(data, root);
+
+            doc.Element("LocalSellers").Add(root);
+            doc.Save(filePath);
+        }
+
+        private static void addProductTypesToXml(LocationData data, XElement root)
```

Comment:
Function names should be in PascalCase.
```suggestion
        private static void AddProductTypesToXml(LocationData data, XElement root)
```